back/app/models/idea.rb

```diff
@@ -245,6 +253,12 @@ def fix_comments_count_on_projects
     Comment.counter_culture_fix_counts only: [%i[idea project]]
   end
 
+  def fix_internal_comments_count_on_projects
+    return unless project_id_previously_changed?
+
+    Comment.counter_culture_fix_counts only: [%i[idea project]]
```

Comment:
I've changed `Comment ` to `InternalComment`, but GitHub failing to pick up pushes at the moment.